src/main/java/io/kabanero/website/TLSFilter.java

```diff
@@ -49,7 +49,7 @@ public void doFilter(ServletRequest req, ServletResponse resp, FilterChain chain
           response.setHeader("X-Frame-Options", "SAMEORIGIN"); // Prevent framing of this website.
           response.setHeader("X-XSS-Protection", "1; mode=block"); // Cross-site scripting prevention for Chrome, Safari, and IE. It's not necessary with newer browser versions that support the Content-Security-Policy but it helps prevent XSS on older versions of these browsers.
           response.setHeader("X-Content-Type-Options", "nosniff"); // Stops a browser from trying to MIME-sniff the content type.
-          response.setHeader("Content-Security-Policy", "default-src 'self' 'unsafe-inline' marketplace.eclipse.org maxcdn.bootstrapcdn.com fonts.googleapis.com ajax.googleapis.com fonts.gstatic.com  *.githubusercontent.com api.github.com www.googletagmanager.com tagmanager.google.com www.google-analytics.com cdnjs.cloudflare.com *.twitter.com *.twimg.com https://www.youtube.com data:"); // Mitigating cross site scripting (XSS) from other domains.
+          response.setHeader("Content-Security-Policy", "default-src 'self' 'unsafe-inline' marketplace.eclipse.org maxcdn.bootstrapcdn.com fonts.googleapis.com ajax.googleapis.com fonts.gstatic.com  *.githubusercontent.com api.github.com www.googletagmanager.com translate.googleapis.com translate.google.com tagmanager.google.com www.google-analytics.com cdnjs.cloudflare.com *.twitter.com *.twimg.com https://www.youtube.com data:"); // Mitigating cross site scripting (XSS) from other domains.
```

Comment:
You'll need to add another exception here for `gstatic.com`
>(index):1 Refused to load the image 'https://www.gstatic.com/images/branding/product/1x/translate_24dp.png' because it violates the following Content Security Policy directive: ........